packages/dbml-core/src/export/DbmlExporter.js

```diff
@@ -263,59 +266,67 @@ class DbmlExporter {
     return strArr.length ? strArr.join('\n') : '';
   }
 
+  static getTableGroupSettings (tableGroup) {
+    const settings = [];
+
+    if (tableGroup.color) settings.push(`color: ${tableGroup.color}`);
+
+    return settings.length ? ` [${settings.join(', ')}]` : '';
+  }
+
   static exportTableGroups (tableGroupIds, model) {
     const tableGroupStrs = tableGroupIds.map(groupId => {
       const group = model.tableGroups[groupId];
       const groupSchema = model.schemas[group.schemaId];
+      const groupSettingStr = DbmlExporter.getTableGroupSettings(group);
 
-      return `TableGroup ${shouldPrintSchema(groupSchema, model)
-        ? `"${groupSchema.name}".` : ''}"${group.name}" {\n${
-        group.tableIds.map(tableId => {
+      const groupNote = group.note ? `  Note: ${DbmlExporter.escapeNote(group.note)}\n`: '';
+      const groupSchemaName = shouldPrintSchema(groupSchema, model) ? `"${groupSchema.name}".` : '';
+      const groupName = `${groupSchemaName}"${group.name}"`;
+
+      const tableStr = group.tableIds
+        .map(tableId => {
           const table = model.tables[tableId];
           const tableSchema = model.schemas[table.schemaId];
           return `  ${shouldPrintSchema(tableSchema, model)
-            ? `"${tableSchema.name}".` : ''}"${table.name}"`;
-        }).join('\n')}\n}\n`;
+            ? `"${tableSchema.name}".` : ''}"${table.name}"\n`;
+        })
+        .join('');
+
+      return `TableGroup ${groupName}${groupSettingStr} {\n${tableStr}${groupNote}}\n`;
     });
 
     return tableGroupStrs.length ? tableGroupStrs.join('\n') : '';
   }
 
+  static exportStickyNotes (model) {
+    const noteStrs = [];
+    forIn(model.notes, (note) => {
+      const escapedContent = `  ${DbmlExporter.escapeNote(note.content)}`;
+      noteStrs.push(`Note ${note.name} {\n${escapedContent}\n}\n`);
+    });
+    return noteStrs.join('\n');
```

Comment:
Instead of writing the code like this. Use can use `reduce` from lodash to create the sticky notes string.

Reply:
It would be something like this. Please re-check.

```javascript
import { reduce } from 'lodash';

static exportStickyNotes (model) {
  return reduce(model.notes, (stickyNotes, note) => {
    const escapedContent = ` ${DbmlExporter.escapeNote(note.content)}`;
    const stickyNote =  `Note ${note.name} {\n${escapedContent}\n}\n`;
    stickyNotes += `${stickyNote}\n`;
    return stickyNotes;
  }, '');
};
```

Reply:
I updated the code using lodash's `reduce` nha a